Can't use generated bitflags to initialise a const

This works in plain Rust

```rust
const MAMMAL: u8 = 1;
const AQUATIC: u8 = 2;
const SEAL: u8 = MAMMAL | AQUATIC;
```

but the equivalent when using bitflags does not

```rust
use bitflags::bitflags;

bitflags! {
    pub struct FLAGS: u8 {
        const MAMMAL  = 1;
        const AQUATIC = 2;
    }
}

const SECOND_SEAL: FLAGS = FLAGS::MAMMAL | FLAGS::AQUATIC;
```

You get an error *error[E0015]: calls in constants are limited to constant functions, tuple structs and tuple variants* when trying to BITOR the two flags.

It would be nice if it did, because in my program I have a large initial array of flags to create and there are many different combinations making it impractical to spell them all out inside the `bitflags!` call.

Edit: Simplified the repro.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+# 1.1.0
+
+This is a re-release of `1.0.5`, which was yanked due to a bug in the RLS.
+
 # 1.0.5
 
 - Use compiletest_rs flags supported by stable toolchain ([#171])

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4,7 +4,7 @@ name = "bitflags"
 # NB: When modifying, also modify:
 #   1. html_root_url in lib.rs
 #   2. number in readme (for breaking changes)
-version = "1.0.5"
+version = "1.1.0"
 authors = ["The Rust Project Developers"]
 license = "MIT/Apache-2.0"
 keywords = ["bit", "bitmask", "bitflags", "flags"]

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -247,7 +247,7 @@
 //! ```
 
 #![no_std]
-#![doc(html_root_url = "https://docs.rs/bitflags/1.0.5")]
+#![doc(html_root_url = "https://docs.rs/bitflags/1.1.0")]
 
 #[cfg(test)]
 #[macro_use]

